vars/runPipeline.groovy

```diff
@@ -456,12 +460,13 @@ def call() { timeout(time: 1, unit: 'HOURS') {
         "libmir/mir",
         // sorted by test time fast to slow (to minimize pending queue length)
         "dlang-community/dfmt", // 11s
+        "BBasile/iz", //12s
```

Comment:
Also I had to be a style facist, but all the other tests use `// Xs` ...
(I would usually fix such nits myself)

Reply:
if so you would have made your own check in dscanner for the style.